moveit_core/CMakeLists.txt

```diff
@@ -183,6 +178,19 @@ install(
 )
 
 ament_export_targets(export_${PROJECT_NAME} HAS_LIBRARY_TARGET)
+
+install(
+  TARGETS
+    collision_detector_bullet_plugin
+    collision_detector_fcl_plugin
```

Comment:
For some reason, `collision_detector_fcl_plugin` was actually not in the previous list of targets to install -- but it builds fine.

@henningkayser do you maybe know if there was something specific with Humble where we shouldn't be building it?